playwright-tests/playwright.config.ts

```diff
@@ -0,0 +1,34 @@
+import { PlaywrightTestConfig } from 'playwright/test';
+import { REEF_KNOT_CONFIG, REPORT_CONFIG } from '@config';
+
+const config: PlaywrightTestConfig = {
+  testDir: './tests',
+  timeout: 5 * 60 * 1000,
+  expect: {
+    timeout: 5000,
+  },
+  fullyParallel: false,
+  forbidOnly: !!process.env.CI,
+  retries: 1,
+  workers: 1,
+  reporter: REPORT_CONFIG(),
```

Comment:
REPORT_CONFIG function must be a lowercase

Reply:
fixed